plugins/modules/system/filesystem.py

```diff
@@ -30,25 +32,30 @@
     default: present
     version_added: 1.3.0
   fstype:
-    choices: [ btrfs, ext2, ext3, ext4, ext4dev, f2fs, lvm, ocfs2, reiserfs, xfs, vfat, swap ]
+    choices: [ btrfs, ext2, ext3, ext4, ext4dev, f2fs, lvm, ocfs2, reiserfs, xfs, vfat, swap, ufs ]
     description:
       - Filesystem type to be created. This option is required with
         C(state=present) (or if I(state) is omitted).
       - reiserfs support was added in 2.2.
       - lvm support was added in 2.5.
-      - since 2.5, I(dev) can be an image file.
-      - vfat support was added in 2.5
-      - ocfs2 support was added in 2.6
-      - f2fs support was added in 2.7
-      - swap support was added in 2.8
+      - vfat support was added in 2.5.
+      - ocfs2 support was added in 2.6.
+      - f2fs support was added in 2.7.
+      - swap support was added in 2.8.
```

Comment:
Can you remove the `x.y` version numbers completely? These refer to Ansible versions (without saying so), and have all been supported since community.general 0.1.0. And if someone really wants to know about support in older Ansible versions, they can look in the Ansible 2.9 docs...

Reply:
Yes, I can. Moreover, I found them confusing.

Reply:
Thanks! They are indeed, especially if `Ansible` isn't mentioned so that people know what these versions refer to :)